### DISCUSSION TITLE
Portal or FAB.Group over navigator

### DISCUSSION THREAD

**Discussion by tomaszukovskij** [callstack/react-native-paper#3918] [Q&A]
Hi,

I have tried the FAB.Group to have the actions with overlay over whole screen.
I have nested navigation where I need to have a FAB. But implementing everything from docs, I have a view like. Is there any way to have a overlay other navigation? 
Thanks
![image](https://github.com/callstack/react-native-paper/assets/25223364/4d7d2a1d-7507-43f1-84c9-d9c6969cd5ce)

**Comment by lukewalczak** [comment]
Wrapping `FAB.Group` into `Portal` should fix your problem.
